poincare/src/least_common_multiple.cpp

```diff
@@ -49,7 +49,7 @@ Evaluation<T> * LeastCommonMultiple::templatedEvaluate(Context& context, AngleUn
     a = b;
     b = r;
   }
-  return new Complex<T>(Complex<T>::Float(std::round((T)(product/a))));
+  return new Complex<T>(Complex<T>::Float(product/a));
```

Comment:
Is this what's causing the travis blackbox build to fail?

Reply:
No, it was a wrong assertion I added in poincare/src/integer.cpp ... Thank you for helping with coverity!

Reply:
Ah, right. And no problem, Coverity is a very useful tool I've used on other projects, so I'm glad you find it helpful too :)